### ISSUE TITLE
Transition between pages

### ISSUE THREAD

**Issue by AryanNanda17** [AryanNanda17/NonStop-Lofi-Vibe-Generator#50]
Our pages currently transition abruptly, which impacts the user experience. Adding smooth page transitions would improve the interface's flow and make navigation more seamless.

**Comment by AryanNanda17** [comment]
Contributors can take inspiration from  [this](https://www.palantir.com/) site.

**Comment by Badra00** [comment]
Hey 
Could you assign this to me?

**Comment by ParamThakkar123** [comment]
@Badra00 Yes. You can work on this. I will assign this to you. Thanks for your interest in working on our project !

**Comment by AryanNanda17** [comment]
@Badra00, I have unassigned you from this issue due to lack of updates.

**Comment by RNAdvani** [comment]
@AryanNanda17  please assign this to me

**Comment by ParamThakkar123** [comment]
@RNAdvani Assigned this issue to you

**Comment by AryanNanda17** [comment]
@RNAdvani, also it would be really great and motivating for our team if you could star this repo so that this project reaches more people. Thanks again for your interest. I am assigning this to you

**Comment by RNAdvani** [comment]
yes i would fork and star the repo, also could you add labels "gssoc-ext" and any level of your convenience, it would be great

**Comment by ParamThakkar123** [comment]
Yes. We will do that for sure

**Comment by RNAdvani** [comment]
#72 added page transition
